src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/RelatedAssemblyAttribute.cs

```diff
@@ -107,5 +112,15 @@ internal static IReadOnlyList<Assembly> GetRelatedAssemblies(Assembly assembly,
 
             return relatedAssemblies;
         }
+
+        internal static string GetAssemblyLocation(Assembly assembly)
```

Comment:
Need to parse this as a Uri when this is shadow copied. For all other cases, CodeBase == Location.